### PR TITLE
R1 jerry

### PR DESCRIPTION
**Pull request by 348428118**
Update the factory-test functions, including shutdown, setvolume, usbled turn off,  standby modeand command list

### CODE REVIEW

File: r1/set_usbled_off.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#turn off usb led 
+adk-message-send 'led_start_pattern {pattern:33}'
```
Comment (Alexey-Abdulov-Tymphany) [inline r1/set_usbled_off.sh:2]: How does PT deal with pulsing PLAY/PAUSE? Do they also call a script from system?

Comment (348428118) [reply r1/set_usbled_off.sh:2]: My plan is to add this to command list, they can search and execute it to turn off.  I tested it works

Comment (348428118) [reply r1/set_usbled_off.sh:2]: I will talk with Fancy, to check his feedback for this method.

Comment (Alexey-Abdulov-Tymphany) [reply r1/set_usbled_off.sh:2]: How does PT deal with pulsing PLAY/PAUSE?

**Review by Alexey-Abdulov-Tymphany — APPROVED** [review]

**Pull request merged**